### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by podcastinator**
1. Add contact details
2. Update description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,9 @@ Please feel free to reach out to us if you would like use Secure XGBoost for you
 ## Documentation
 
 To get started with the library, please refer to the [documentation](https://secure-xgboost.readthedocs.io/en/latest/).
+
+## Contact
+If you would like to know more about our project or have questions, please contact us at:
+* Rishabh Poddar (rishabhp@eecs.berkeley.edu)
+* Chester Leung (chester.leung@berkeley.edu)
+* Wenting Zheng (wzheng@berkeley.edu)
```
Comment (chester-leung) [inline README.md]: I think Wenting's email is wzheng13@berkeley.edu.

Comment (podcastinator) [reply README.md]: Whoops.

Comment (chester-leung) [reply README.md]: Can you change my email to eecs too? <chester@eecs.berkeley.edu>

**Review by chester-leung — APPROVED** [review]

**Comment by chester-leung** [comment]
Thanks!

**Pull request merged**